src/common/templates/_ingress.tpl

```diff
@@ -26,6 +26,9 @@ metadata:
     {{- if $.Values.global.ingress.objects.annotations }}
     {{- include "harnesscommon.tplvalues.render" (dict "value" $.Values.global.ingress.objects.annotations "context" $) | nindent 4 }}
     {{- end }}
+    {{- if $.Values.ingress.serviceUpStream.enabled }}
```

Comment:
hmm, not like this. make it controllable via values.yaml 
when you provide an annotation you can wrap it under a condition.
also bump up the chart version in Chart.yaml